test/api_spec.js

```diff
@@ -852,7 +851,7 @@ describe("api", function () {
   });
   // For this test to work, "Auto-create folders" should be enabled in the Upload Settings.
   // Replace `it` with  `it.skip` below if you want to disable it.
```

Comment:
Why are we skipping all of these tests? Should we not make sure that these clouds have all the needed settings?

Reply:
Some of our tests have big assumptions about the cloud and account we're testing against. 

Since fixing all these tests is out of scope this PR, we'll skip invalid tests and fix them in near-future PRs